apps/asap-cli/src/import/users/insert.ts

```diff
@@ -61,6 +61,9 @@ const insertTeam = async (data: Data, cache: Cache): Promise<RestTeam> => {
     applicationNumber: {
       iv: application,
     },
+    active: {
```

Comment:
I just hard coded this true here, but I am not sure if that's the right thing to do 🤔 I saw that above we have

```
  const team = {
    displayName: {
      iv: lastName,
    },
```

which is kind of weird that the team name is the person's last name. I am assuming that the data here is not important.

Reply:
meh, we dont really use this import anymore anyway, so